api/room/schema.py

```diff
@@ -85,17 +85,22 @@ def mutate(self, info, office_id, **kwargs):
         validate_empty_fields(**kwargs)
         verify_ids(office_id, kwargs)
         get_office = Office.query.filter_by(id=office_id).first()
-        if not get_office:
```

Comment:
This piece of code has been removed because it is redundant.
The function `verify_ids` already performs the check to see if that office exists or not.